mkdocs/nav.py

```diff
@@ -148,6 +186,63 @@ def __init__(self, title, url, path, url_context):
         self.next_page = None
         self.ancestors = []
 
+        self.markdown, self.meta = self.load_markdown()
+
+    def __eq__(self, other):
```

Comment:
It feels a bit like a hack adding this, but it made the tests much easier to do. Need to decide if we want to keep it or not.